Tests/ZippyJSONTests/ZippyJSONDecoderTests.swift

```diff
@@ -384,7 +384,7 @@ class ZippyJSONTests: XCTestCase {
             let a: [String: String]
         }
         // testRoundTrip(of: Aa.self, json: #"{"a": {}}"#)
-        _testFailure(of: Aa.self, json: #"{"a": 2}"#)
+        _testFailure(of: Aa.self, json: #"{"a": 2}"#, relaxedErrorCheck: true)
```

Comment:
Had to do it because now zippyJSONDecoder and regular JSONDecoder hit different code paths for dictionaries